app/helpers/recent_items_helper.rb

```diff
@@ -0,0 +1,89 @@
+# A random selection of recently modified works to display as thumbnails
```

Comment:
Nice job using actual object-oriented design (the RecentItems class/object), instead of just a bunch of flat helper methods, like Rails tries to get you to. 

There's really no reason this _needs_ to be in ./app/helpers -- the _standard_ thing to be in ./app/helpers would be that formentioned flat list of Rails 'helper' methods (which get mixed into the 'view context').  

But I don't see any reason it would cause a problem, and don't know any reason _not_ to put it here, so that's fine.